### ISSUE TITLE
New submissions for Thu, 30 Jul 20

### ISSUE THREAD

**Issue by kobiso** [kobiso/daily-arxiv-noti#137]
## Keyword: metric learning
There is no result 
## Keyword: image retrieval
There is no result 
## Keyword: face recognition
There is no result 
### Meta-LR-Schedule-Net: Learned LR Schedules that Scale and Generalize
 - **Authors:** Jun Shu, Yanwen Zhu, Qian Zhao, Deyu Meng, Zongben Xu
 - **Subjects:** Machine Learning (cs.LG); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2007.14546
 - **Pdf link:** https://arxiv.org/pdf/2007.14546
 - **Abstract**
 The learning rate (LR) is one of the most important hyper-parameters in stochastic gradient descent (SGD) for deep neural networks (DNNs) training and generalization. However, current hand-designed LR schedules need to manually pre-specify schedule as well as its extra hyper-parameters, which limits its ability to adapt non-convex optimization problems due to the significant variation of training dynamic. To address this issue, we propose a model capable of adaptively learning LR schedule from data. We specifically design a meta-learner with explicit mapping formulation to parameterize LR schedules, which can adjust LR adaptively to comply with current training dynamic by leveraging the information from past training histories. Image and text classification benchmark experiments substantiate the capability of our method for achieving proper LR schedules compared with baseline methods. Moreover, we transfer the learned LR schedule to other various tasks, like different training batch sizes, epochs, datasets, network architectures, especially large scale ImageNet dataset, showing its stronger generalization capability than related methods. Finally, guided by a small set of clean validation set, we show our method can achieve better generalization error when training data is biased with corrupted noise than baseline methods.
### Dreaming: Model-based Reinforcement Learning by Latent Imagination  without Reconstruction
 - **Authors:** Masashi Okada, Tadahiro Taniguchi
 - **Subjects:** Machine Learning (cs.LG); Artificial Intelligence (cs.AI); Systems and Control (eess.SY); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2007.14535
 - **Pdf link:** https://arxiv.org/pdf/2007.14535
 - **Abstract**
 In the present paper, we propose a decoder-free extension of Dreamer, a leading model-based reinforcement learning (MBRL) method from pixels. Dreamer is a sample- and cost-efficient solution to robot learning, as it is used to train latent state-space models based on a variational autoencoder and to conduct policy optimization by latent trajectory imagination. However, this autoencoding based approach often causes object vanishing, in which the autoencoder fails to perceives key objects for solving control tasks, and thus significantly limiting Dreamer's potential. This work aims to relieve this Dreamer's bottleneck and enhance its performance by means of removing the decoder. For this purpose, we firstly derive a likelihood-free and InfoMax objective of contrastive learning from the evidence lower bound of Dreamer. Secondly, we incorporate two components, (i) independent linear dynamics and (ii) the random crop data augmentation, to the learning scheme so as to improve the training performance. In comparison to Dreamer and other recent model-free reinforcement learning methods, our newly devised Dreamer with InfoMax and without generative decoder (Dreaming) achieves the best scores on 5 difficult simulated robotics tasks, in which Dreamer suffers from object vanishing.
### Learning from Scale-Invariant Examples for Domain Adaptation in Semantic  Segmentation
 - **Authors:** M.Naseer Subhani, Mohsen Ali
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2007.14449
 - **Pdf link:** https://arxiv.org/pdf/2007.14449
 - **Abstract**
 Self-supervised learning approaches for unsupervised domain adaptation (UDA) of semantic segmentation models suffer from challenges of predicting and selecting reasonable good quality pseudo labels. In this paper, we propose a novel approach of exploiting scale-invariance property of the semantic segmentation model for self-supervised domain adaptation. Our algorithm is based on a reasonable assumption that, in general, regardless of the size of the object and stuff (given context) the semantic labeling should be unchanged. We show that this constraint is violated over the images of the target domain, and hence could be used to transfer labels in-between differently scaled patches. Specifically, we show that semantic segmentation model produces output with high entropy when presented with scaled-up patches of target domain, in comparison to when presented original size images. These scale-invariant examples are extracted from the most confident images of the target domain. Dynamic class specific entropy thresholding mechanism is presented to filter out unreliable pseudo-labels. Furthermore, we also incorporate the focal loss to tackle the problem of class imbalance in self-supervised learning. Extensive experiments have been performed, and results indicate that exploiting the scale-invariant labeling, we outperform existing self-supervised based state-of-the-art domain adaptation methods. Specifically, we achieve 1.3% and 3.8% of lead for GTA5 to Cityscapes and SYNTHIA to Cityscapes with VGG16-FCN8 baseline network.
### $S^3$Net: Semantic-Aware Self-supervised Depth Estimation with Monocular  Videos and Synthetic Data
 - **Authors:** Bin Cheng, Inderjot Singh Saggu, Raunak Shah, Gaurav Bansal, Dinesh Bharadia
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2007.14511
 - **Pdf link:** https://arxiv.org/pdf/2007.14511
 - **Abstract**
 Solving depth estimation with monocular cameras enables the possibility of widespread use of cameras as low-cost depth estimation sensors in applications such as autonomous driving and robotics. However, learning such a scalable depth estimation model would require a lot of labeled data which is expensive to collect. There are two popular existing approaches which do not require annotated depth maps: (i) using labeled synthetic and unlabeled real data in an adversarial framework to predict more accurate depth, and (ii) unsupervised models which exploit geometric structure across space and time in monocular video frames. Ideally, we would like to leverage features provided by both approaches as they complement each other; however, existing methods do not adequately exploit these additive benefits. We present $S^3$Net, a self-supervised framework which combines these complementary features: we use synthetic and real-world images for training while exploiting geometric, temporal, as well as semantic constraints. Our novel consolidated architecture provides a new state-of-the-art in self-supervised depth estimation using monocular videos. We present a unique way to train this self-supervised framework, and achieve (i) more than $15\%$ improvement over previous synthetic supervised approaches that use domain adaptation and (ii) more than $10\%$ improvement over previous self-supervised approaches which exploit geometric constraints from the real data.
### What My Motion tells me about Your Pose: Self-Supervised Fine-Tuning of  Observed Vehicle Orientation Angle
 - **Authors:** Cédric Picron, Punarjay Chakravarty, Tom Roussel, Tinne Tuytelaars
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2007.14812
 - **Pdf link:** https://arxiv.org/pdf/2007.14812
 - **Abstract**
 The determination of the relative 6 Degree of Freedom (DoF) pose of vehicles around the ego-vehicle from monocular cameras is an important aspect of the perception problem for Autonomous Vehicles (AVs) and Driver Assist Technology (DAT). Current deep learning techniques used for tackling this problem are data hungry, driving the need for unsupervised or self-supervised methods. In this paper, we consider the domain adaptation task of fine-tuning a vehicle orientation estimator on a new domain without labels. By leveraging the ego-motion consistencies obtained from a monocular SLAM method, we show that our self-supervised fine-tuning scheme consistently improves the accuracy of the resulting network. More specifically, when transitioning from Virtual Kitti to nuScenes, up to 70% of the performance is recovered compared to the 100% of a supervised method. Our self-supervised method hence allows us to safely transfer vehicle orientation estimators to new domains without requiring expensive new labels.
### Learning Video Representations from Textual Web Supervision
 - **Authors:** Jonathan C. Stroud, David A. Ross, Chen Sun, Jia Deng, Rahul Sukthankar, Cordelia Schmid
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2007.14937
 - **Pdf link:** https://arxiv.org/pdf/2007.14937
 - **Abstract**
 Videos found on the Internet are paired with pieces of text, such as titles and descriptions. This text typically describes the most important content in the video, such as the objects in the scene and the actions being performed. Based on this observation, we propose to use such text as a method for learning video representations. To accomplish this, we propose a data collection process and use it to collect 70M video clips shared publicly on the Internet, and we then train a model to pair each video with its associated text. We fine-tune the model on several down-stream action recognition tasks, including Kinetics, HMDB-51, and UCF-101. We find that this approach is an effective method of pretraining video representations. Specifically, it leads to improvements over from-scratch training on all benchmarks, outperforms many methods for self-supervised and webly-supervised video representation learning, and achieves an improvement of 2.2% accuracy on HMDB-51.
### Almost exact recovery in noisy semi-supervised learning
 - **Authors:** Konstantin Avrachenkov, Maximilien Dreveton
 - **Subjects:** Machine Learning (cs.LG); Statistics Theory (math.ST); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2007.14717
 - **Pdf link:** https://arxiv.org/pdf/2007.14717
 - **Abstract**
 This paper investigates noisy graph-based semi-supervised learning or community detection. We consider the Stochastic Block Model (SBM), where, in addition to the graph observation, an oracle gives a non-perfect information about some nodes' cluster assignment. We derive the Maximum A Priori (MAP) estimator, and show that a continuous relaxation of the MAP performs almost exact recovery under non-restrictive conditions on the average degree and amount of oracle noise. In particular, this method avoids some pitfalls of several graph-based semi-supervised learning methods such as the flatness of the classification functions, appearing in the problems with a very large amount of unlabeled data.
### An Adaptive Educational eLearning System based on Semantics, Ontologies  Matching and Recomentation System
 - **Authors:** Vasiliki Demertzi, Konstantinos Demertzis
 - **Subjects:** Information Retrieval (cs.IR)
 - **Arxiv link:** https://arxiv.org/abs/2007.14771
 - **Pdf link:** https://arxiv.org/pdf/2007.14771
 - **Abstract**
 The implementation of teaching interventions in learning needs has received considerable attention, as the provision of the same educational conditions to all students, is pedagogically ineffective. In contrast, more effectively considered the pedagogical strategies that adapt to the real individual skills of the students. An important innovation in this direction is the Adaptive Educational Systems (AES) that support automatic modeling study and adjust the teaching content on educational needs and students' skills. Effective utilization of these educational approaches can be enhanced with Artificial Intelligence (AI) technologies in order to the substantive content of the web acquires structure and the published information is perceived by the search engines. This study proposes a novel Adaptive Educational eLearning System (AEeLS) that has the capacity to gather and analyze data from learning repositories and to adapt these to the educational curriculum according to the student skills and experience. It is a novel hybrid machine learning system that combines a Semi-Supervised Classification method for ontology matching and a Recommendation Mechanism that uses a hybrid method from neighborhood-based collaborative and content-based filtering techniques, in order to provide a personalized educational environment for each student.
### Computing Weighted Subset Transversals in $H$-Free Graphs
 - **Authors:** Nick Brettell, Matthew Johnson, Daniel Paulusma
 - **Subjects:** Data Structures and Algorithms (cs.DS)
 - **Arxiv link:** https://arxiv.org/abs/2007.14514
 - **Pdf link:** https://arxiv.org/pdf/2007.14514
 - **Abstract**
 For the Odd Cycle Transversal problem, the task is to find a small set $S$ of vertices in a graph that intersects every cycle of odd length. The generalization Subset Odd Cycle Transversal requires that $S$ only intersects those odd cycles that include a vertex of a distinguished subset $T$ of the vertex set. If we are also given weights for the vertices of the graph, we can ask instead that $S$ has small weight: this is the problem Weighted Subset Odd Cycle Transversal. We prove an almost-complete complexity dichotomy for this problem when the input is restricted to graphs that do not contain a graph~$H$ as an induced subgraph. In particular, we show that for $(3P_1+P_2)$-free graphs (where $P_r$ is the path on $r$ vertices) there is a polynomial-time algorithm, but the problem is NP-complete for $5P_1$-free graphs, that is, graphs of independence number~$4$. Thus we obtain a dichotomy with respect to the independence number; this is an analogue of the dichotomy for Weighted Subset Feedback Vertex Set recently obtained by Papadopoulos and Tzimas. In contrast, Subset Feedback Vertex Set and Subset Odd Cycle Transversal have a polynomial-time algorithm for any graph class with bounded independence number. We also generalize the polynomial-time result of Papadopoulos and Tzimas for Weighted Subset Feedback Vertex Set on $4P_1$-free graphs to $(3P_1+P_2)$-free graphs. As a consequence, we show that the complexity for both of the weighted subset transversal problems restricted to $H$-free graphs remains open for just three particular graphs~$H$.
### Meta-LR-Schedule-Net: Learned LR Schedules that Scale and Generalize
 - **Authors:** Jun Shu, Yanwen Zhu, Qian Zhao, Deyu Meng, Zongben Xu
 - **Subjects:** Machine Learning (cs.LG); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2007.14546
 - **Pdf link:** https://arxiv.org/pdf/2007.14546
 - **Abstract**
 The learning rate (LR) is one of the most important hyper-parameters in stochastic gradient descent (SGD) for deep neural networks (DNNs) training and generalization. However, current hand-designed LR schedules need to manually pre-specify schedule as well as its extra hyper-parameters, which limits its ability to adapt non-convex optimization problems due to the significant variation of training dynamic. To address this issue, we propose a model capable of adaptively learning LR schedule from data. We specifically design a meta-learner with explicit mapping formulation to parameterize LR schedules, which can adjust LR adaptively to comply with current training dynamic by leveraging the information from past training histories. Image and text classification benchmark experiments substantiate the capability of our method for achieving proper LR schedules compared with baseline methods. Moreover, we transfer the learned LR schedule to other various tasks, like different training batch sizes, epochs, datasets, network architectures, especially large scale ImageNet dataset, showing its stronger generalization capability than related methods. Finally, guided by a small set of clean validation set, we show our method can achieve better generalization error when training data is biased with corrupted noise than baseline methods.
### New Constructions of Optimal Cyclic (r,δ) Locally Repairable Codes  from Their Zeros
 - **Authors:** Jing Qiu, Dabin Zheng, Fang-Wei Fu
 - **Subjects:** Information Theory (cs.IT)
 - **Arxiv link:** https://arxiv.org/abs/2007.14752
 - **Pdf link:** https://arxiv.org/pdf/2007.14752
 - **Abstract**
 An $(r, \delta)$-locally repairable code ($(r, \delta)$-LRC for short) was introduced by Prakash et al. \cite{Prakash2012} for tolerating multiple failed nodes in distributed storage systems, which was a generalization of the concept of $r$-LRCs produced by Gopalan et al. \cite{Gopalan2012}. An $(r, \delta)$-LRC is said to be optimal if it achieves the Singleton-like bound. Recently, Chen et al. \cite{Chen2018} generalized the construction of cyclic $r$-LRCs proposed by Tamo et al. \cite{Tamo2015,Tamo2016} and constructed several classes of optimal $(r, \delta)$-LRCs of length $n$ for $n\, |\, (q-1)$ or $n\,|\, (q+1)$, respectively in terms of a union of the set of zeros controlling the minimum distance and the set of zeros ensuring the locality. Following the work of \cite{Chen2018,Chen2019}, this paper first characterizes $(r, \delta)$-locality of a cyclic code via its zeros. Then we construct several classes of optimal cyclic $(r, \delta)$-LRCs of length $n$ for $n\, |\, (q-1)$ or $n\,|\, (q+1)$, respectively from the product of two sets of zeros. Our constructions include all optimal cyclic $(r,\delta)$-LRCs proposed in \cite{Chen2018,Chen2019}, and our method seems more convenient to obtain optimal cyclic $(r, \delta)$-LRCs with flexible parameters. Moreover, many optimal cyclic $(r,\delta)$-LRCs of length $n$ for $n\, |\, (q-1)$ or $n\,|\, (q+1)$, respectively such that $(r+\delta-1)\nmid n$ can be obtained from our method.
### Translate the Facial Regions You Like Using Region-Wise Normalization
 - **Authors:** Wenshuang Liu, Wenting Chen, Linlin Shen
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2007.14615
 - **Pdf link:** https://arxiv.org/pdf/2007.14615
 - **Abstract**
 Though GAN (Generative Adversarial Networks) based technique has greatly advanced the performance of image synthesis and face translation, only few works available in literature provide region based style encoding and translation. We propose in this paper a region-wise normalization framework, for region level face translation. While per-region style is encoded using available approach, we build a so called RIN (region-wise normalization) block to individually inject the styles into per-region feature maps and then fuse them for following convolution and upsampling. Both shape and texture of different regions can thus be translated to various target styles. A region matching loss has also been proposed to significantly reduce the inference between regions during the translation process. Extensive experiments on three publicly available datasets, i.e. Morph, RaFD and CelebAMask-HQ, suggest that our approach demonstrate a large improvement over state-of-the-art methods like StarGAN, SEAN and FUNIT. Our approach has further advantages in precise control of the regions to be translated. As a result, region level expression changes and step by step make up can be achieved. The video demo is available at https://youtu.be/ceRqsbzXAfk.
